releaseNotes/JavaScriptProvider.md

```diff
@@ -0,0 +1,27 @@
+# Azure App Configuration JavaScript Provider
+
+[Source code][source_code] | [Package (npm)][package] | [Samples][samples]
+
+## 1.0.0-preview.2 - December 15, 2023
+### Breaking Changes
+- The label filter in a selector is restricted to a single label. An error will be thrown if a label filter contains `*` or `,`. This change is to avoid the ambiguity when multiple values are loaded for the same key. Key-values with different labels can still be loaded using separate selectors for proper configuration composition. [#22](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/22)
+
+### Bug Fixes
+- When multiple selectors are provided, the last one takes precedence. If there are selectors with the same key filter and label filter, they are deduplicated. This change ensures that the original selector precedence is maintained as provided. [#23](https://github.com/Azure/AppConfiguration-JavaScriptProvider/issues/23)

```

Comment:
This seems more like an internal improvement. Functionally, it has no difference to customers. Did I miss anything?

Reply:
To be accurate, it's rather a breaking change of the behavior.

When you specify providers as: 
```
selectors: [ 
            { keyFilter: "app.*", labelFilter: "dev" },
            { keyFilter: "app.*", labelFilter: "prod" },
            { keyFilter: "app.*", labelFilter: "dev" },
] 
```
Previous behavior: label `prod` wins.
Current behavior: label `dev` wins.

Reply:
Because previous behavior is not as expected, so I place this item under "Bug Fixes" category. Do you think we should explicitly announce it as a breaking change?

Reply:
Yes, indeed, this should go to the breaking change section. We should also include information about what customers have to do. Does this make sense?

Fixed a bug where the last selector may not take precedence if multiple selectors with the same key and label filters are provided. After applying the fix, the resulting configuration of an application may change. To ensure the intended configuration composition, remove any duplicated selectors.

Reply:
looks good to me